cocos/core/platform/sys.ts

```diff
@@ -23,16 +23,18 @@
  THE SOFTWARE.
 */
 
+import { BYTEDANCE, WECHAT, WECHAT_MINI_PROGRAM } from 'internal:constants';
```

Comment:
It seems these constants are not used.

Reply:
I just moved it from bellow lines. It's better to put constants at the beginning.
And they are used in this file, see
```ts
if (WECHAT || WECHAT_MINI_PROGRAM || BYTEDANCE) {
                    this.__isWebIOS14OrIPadOS14Env = (sys.os === OS.IOS || sys.os === OS.OSX) && GameGlobal?.isIOSHighPerformanceMode
            && /(OS 1((4\.[0-9])|(5\.[0-3])))|(Version\/1((4\.[0-9])|(5\.[0-3])))/.test(window.navigator.userAgent);
                } else {
                    this.__isWebIOS14OrIPadOS14Env = (sys.os === OS.IOS || sys.os === OS.OSX) && systemInfo.isBrowser
            && /(OS 14)|(Version\/14)/.test(window.navigator.userAgent);
                }
```